index.html

```diff
@@ -291,20 +291,20 @@ <h4 class="refer-link">LET'S CONNECT</h4>
                     <h1 class="contact-title-1">Contact me</h1> 
                     <p class="contact-description-1">
                         If you have an application you are interested in developing, a
-                        feature that you need built or a project that needs coding.I’d
+                        feature that you need built or a project that needs coding. I’d
                         love to help with it.
                     </p>
                 </div>
                 <form id="input-contact-box" action="https://formspree.io/f/xpzbqlab" method="post">
                     <label class="label" for="username"></label>
                     <input id="username" class="input-i" name="username" type="text" maxlength="30"
-                        placeholder="Enobong" required="required" />
+                        placeholder="Enobong" required />
                     <label class="label" for="email"></label>
                     <input id="email" class="input-i" name="email" type="email" value="" placeholder="misong4real@yahoo.com"
-                        required="required" />
+                        required />
                     <label class="label" id="input-message" for="msg"></label>
                     <textarea class="input-m" name="msg" id="msg" maxlength="500" cols="20" rows="10"
-                        placeholder="Write your message here" required="required"></textarea>
+                        placeholder="Write your message here" required></textarea>
                     <button class=" get-touch-btn" type="submit">Get In Touch</button>               
                    </form>
```

Comment:
- [ ] [OPTIONAL] Your contact section could be a lot more aesthetic if you can work on adding the `background-image` that is in this section in the wireframe (previous images shared with you by my colleague). This shows your skills as a sterling designer. 😉 

https://rawcdn.githack.com/Enoisong/Mobile-First-Portfolio/0398a79314c7c609a62a1aac7b3da7bc56ab0d54/index.html